x-pack/legacy/plugins/siem/public/components/open_timeline/types.ts

```diff
@@ -36,6 +36,10 @@ export interface TimelineActionsOverflowColumns {
   } | null>;
 }
 
+enum TimelineTypes {
+  default = 'default',
+  template = 'template',
+}
 /** The results of the query run by the OpenTimeline component */
 export interface OpenTimelineResult {
```

Comment:
I am also wondering if some of these types can be only from the common folder. Maybe we can go with a different PR to clean up some types.